test/runtests.jl

```diff
@@ -9,7 +9,7 @@ end
 output = @capture_err begin
     warn("should get captured, not printed")
 end
-@test output == "WARNING: should get captured, not printed\n"
+@test output == "\e[1m\e[31mWARNING: should get captured, not printed\e[0m\n"
```

Comment:
this should probably actually check `Base.have_color` so it can pass in either setting